src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -703,6 +703,7 @@ def test_containerapp_up_dapr_e2e(self, resource_group):
 
 
 class ContainerappServiceBindingTests(ScenarioTest):
+    @live_only()
```

Comment:
May I ask why this test is marked as live only?

Reply:
was! @Greedygre put me on to how to sync changes from azure-cli repo so the tests are back to recording state @yanzhudd please take a look. thanks for the prompt review!!